shared/js/background/tracker-utils.js

```diff
@@ -0,0 +1,81 @@
+// Utility functions for dealing with tracker information
+const utils = require('./utils.es6')
+const trackers = require('./trackers.es6')
+const tldts = require('tldts')
+const tdsStorage = require('./storage/tds.es6')
+
+// Determine if two URL's belong to the same entity.
+function isSameEntity (url1, url2) {
```

Comment:
I agree. It's here because putting it into utils creates a circular dependency, utils is a dependency of trackers, and trackers is a dependency for this function.

Reply:
Ah, I see, that makes sense.